packages/composite-checkout/src/components/stripe-credit-card-fields.js

```diff
@@ -13,11 +13,46 @@ import GridRow from './grid-row';
 import { useLocalize } from '../lib/localize';
 import { useStripe } from '../lib/stripe';
 import { useCheckoutHandlers } from '../index';
+import theme from '../theme';
```

Comment:
We can actually access the current theme object itself from inside the component, which will also support changes if the host page overrides the theme:

```js
import { useContext } from 'react';
import { ThemeContext } from 'styled-components';
// ...
function StripeCreditCardFields() {
  // ...
  const theme = useContext( ThemeContext );
```

(There's even a `useTheme` hook that's [just been added](https://github.com/styled-components/styled-components/pull/2765) but probably isn't available to us yet; but it does the same thing as this.)

Reply:
oh nice! I didn't know that. I've updated the code to use the Theme context.